pkg/testrunner/result/collect.go

```diff
@@ -63,6 +65,27 @@ func (c *Collector) Collect(log logr.Logger, tmClient kubernetes.Interface, name
 			}
 		}
 
+		// upload testrun status to github component
+		if cfg.GithubComponentForStatus != "" {
```

Comment:
maybe we should be more explicit and add an `UploadStatusAsset bool` parameter.
WDYT?